### PR TITLE
Change the PUT and POST method in the tutorials

### PR DESCRIPTION
**Pull request by danivideda**
The convention of the POST method in HTTP should be for creating a new resource. And PUT method for updating specific resource

### CODE REVIEW

**Comment by danivideda** [comment]
Pull-request on the 'starter code' to update the Swagger UI spec: https://github.com/redis-developer/express-redis-om-workshop/pull/2#issue-1421900271

**Review by leibale — APPROVED** [review]

File: docs/stack/get-started/tutorials/stack-node.md [hunk 1]
```diff
@@ -342,7 +342,7 @@ Now that we can read and write, let's implement the *REST* of the HTTP verbs. RE
 Let's add the code to update a person using a POST route:
```
Comment (guyroyse) [inline docs/stack/get-started/tutorials/stack-node.md:342]: Need to update the prose to match the code.

File: docs/stack/get-started/tutorials/stack-node.md [hunk 2]
```diff
@@ -283,7 +283,7 @@ Now we can add our routes to create, read, update, and delete persons. Head back
 We'll create a person first as you need to have persons in Redis before you can do any of the reading, writing, or removing of them. Add the PUT route below. This route will call `.createAndSave()` to create a `Person` from the request body and immediately save it to the Redis:
```
Comment (guyroyse) [inline docs/stack/get-started/tutorials/stack-node.md:283]: Need to update the prose to match the code.

**Review by guyroyse — COMMENTED** [review]
See inline comments.

**Pull request closed without merging**